toc/rfc/rfc-draft-noble-os.md

```diff
@@ -0,0 +1,47 @@
+- The short time available for the overall execution
+- The implementation effort is unknown upfront because it depends on the changes introduced with the new OS version
+- Almost impossible to introduce backwards incompatible changes like removal of unneeded libraries because of the short time available for validation and adoption
+
+
+## Proposal
+
+The CFF community should not skip the Noble Numbat (24.04) release and work on publishing a stemcell line based on it, starting in March or April of 2024. The Slack channel [bosh-noble](https://cloudfoundry.slack.com/archives/C06HTDT78N9) should be used for communication around this effort.
+
+## Workstreams
+
+The major milestones have been documented below in more details per working group.
+
+### Foundational Infrastructure WG
+
+The Foundational Infrastructure Working Group is responsible for making new stemcells. As soon as Noble Numbat (24.04) packages from Ubuntu are available the WG can start working on the new stemcell. The effort in the FI WG includes following rough list of steps:
+- Extend [bosh-linux-stemcell-builder](https://github.com/cloudfoundry/bosh-linux-stemcell-builder) for Noble Numbat
+- Noble Numbat stemcell is produce by CI
+- BOSH agent works
+- BOSH Director works
+- BOSH acceptance tests run successfully
+- Make a 0.x beta version of the stemcell available for CFF community validation
+- After successful validation in the community publish 1.x GA stemcell version on [bosh.io](http://bosh.io)
+
+
+The end of planned support for Jammy Jellyfish (22.04) is 06.2027. That is why for Noble Numbat we can take some time to look into topics from the [wish list](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/milestone/1). The RFC suggests targeting a beta stemcell available for the end of Q3 2024, which could be used by the App Runtime Deployment for validation.
+
+### App Runtime Deployments WG
+
+Once initial confidence has been gained by validating the Noble Numbat stemcell against the BOSH director it is time to validate against cf-deployment. Just like with the BOSH director, an automation should be set up to run the CATS test suite against a Cloud Foundry environment which uses the beta 0.x version of the Noble Numbat stemcell provided by the FI WG. GCP will be used as the initial validation environment but if required validation on AWS could be added. Issues discovered by this validation should be distributed to the related CF components. It is their responsibility to address them. If CF components want to have faster feedback on Noble Numbat issues they can add their own validation. When the cf-deployment is successful the FI WG will release the 1.x GA version of the Noble Numbat stemcell.
```

Comment:
It would be good to define success in the context of cf-deployment a bit more. I'm assuming this would mean passing CF Acceptance tests, but maybe there is more (backup / restore, performance?).

Reply:
Additional point which was brought up yesterday during the TOC meeting was about the required support for two stemcell lines until EOF for Jammy. @jochenehret does the wording in the current state covers the validation for the new stemcell which is done in cf-deployment or we need to extend it? 

In FI WG we will discussed how we want to handle this and I will extend the FI part then. And we can add recommendations for other components.

Reply:
I added details about the comments here. For the `cf-deployment` WG I mentioned that the Noble validation should be the same like for Jammy. I think this covers the point made by @rkoster. @jochenehret please check whether the latest ARD section is OK for you.

Reply:
We could add to the ARD section that cf-deployment users will be able to chose between Jammy and Noble stemcell by an ops file. And maybe indicate that the default stemcell would change from Jammy to Noble well before Jammy EOL (maybe even ~2025).